WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -43,8 +43,16 @@ - (BOOL)fb_isVisible
   CGSize screenSize = FBAdjustDimensionsForApplication(app.frame.size, (UIInterfaceOrientation)[XCUIDevice sharedDevice].orientation);
   CGRect screenFrame = CGRectMake(0, 0, screenSize.width, screenSize.height);
   BOOL rectIntersects = CGRectIntersectsRect(self.visibleFrame, screenFrame);
-  BOOL isActionable = CGRectContainsPoint(app.frame, self.hitPoint);
-  return rectIntersects && isActionable;
+  @try {
+    BOOL isActionable = CGRectContainsPoint(app.frame, self.hitPoint);
+    return rectIntersects && isActionable;
+  } @catch (NSException *e) {
+    if ([e.reason containsString:@"attempt to insert nil object"]) {
+      // This is to workaround known XCTest issue https://github.com/facebook/WebDriverAgent/issues/542
+      return rectIntersects;
```

Comment:
Not sure if this is a good idea. That will be inconsistent.
Can we:
1) Log that it happen with some additional info for diagnostics
2) Return hard NO?